text/web-servers/0006-web-servers-rearchitecture.md

```diff
@@ -0,0 +1,270 @@
+# Web Server Buildpack Re-architecture
+
+## Summary
+
+The Nginx and HTTPD buildpacks will be broken into three buildpacks each to
+allow for features to decrease the complexity of each of the component
+buildpacks and to allow for easier use of the underlying dependencies.
+
+## Motivation
+
+Currently Nginx and HTTPD behave as monolithic buildpacks, installing a
+dependency, running configuration, and writing a start command. This is
+antithetical to the Paketo buildpack philosophy as it means that the buildpacks
+are not modular and it is hard to use components of the buildpack easily, such
+as installing just the dependency which is an issue in buildpacks such PHP.
+
+## Implementation
+
+Split each of these buildpacks in three seperate buildpacks
+
+### {web-server}-dist
```

Comment:
```suggestion
### {web-server}
```

Why do we feel the need to tack on `-dist` here?

Reply:
To differentiate it from the current `nginx` or `httpd` buildpacks which have the whole package

Reply:
Would it not be reasonable to keep those names for the distribution buildpacks and bump the major version in the process?

Reply:
@paketo-buildpacks/web-servers-maintainers Can I get a temperature check on this idea?